app/schema/Auth.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "zod";
+
+export default z.object({
+    body: z.object({
+        username: z.string({
+            required_error: "A Username is required.",
+            invalid_type_error: "Username must be a string",
+        }).min(1),
+        password: z.string({
+            required_error: "A Password is required.",
+            invalid_type_error: "Password must be a string",
+        }).min(1),
+    })
+});
```

Comment:
This is good. 

Just keep the error strings consistent. You have punctuation in the required_error but not invalid_type_error. Super nitpicky lol. 

Also change the file name to something more descriptive, preferably something to do with the route. In the future we're going to have many files in this directory, so maybe we should establish a naming convention early. Such as RouterRouteMethod, maybe? So this would be something like AuthLoginPost.ts. This way we can quickly look at the directory and find what we're looking for or know what the schema is likely named when we're importing it.